lib/fizzy/execute.cpp

```diff
@@ -271,12 +264,16 @@ execution_result execute(Instance& instance, FuncIdx function, std::vector<uint6
             pc = label.pc;
             immediates = label.immediate;
 
-            // FIXME: handle arity
-            assert(label.arity == 0);
-
             // When branch is taken, additional stack items must be dropped.
-            assert(stack.size() >= label.stack_height);
-            stack.resize(label.stack_height);
+            assert(stack.size() >= label.stack_height + label.arity);
+            if (label.arity)
```

Comment:
```suggestion
            if (label.arity > 0)
```

Reply:
Why not `!= 0` instead?

Reply:
Probably the most correct one is `== 1`. Maybe I will just add an assert inside.